aten/src/ATen/native/native_functions.yaml

```diff
@@ -5901,8 +6044,20 @@
   dispatch:
     CPU: _local_scalar_dense_cpu
     CUDA: _local_scalar_dense_cuda
+    MPS: _local_scalar_dense_mps
   variants: function
 
+# MPS LSTM implementation
+
+- func: lstm_mps(Tensor input, Tensor[] hx, Tensor[] params, bool has_biases, int num_layers, float dropout, bool train, bool bidirectional, bool batch_first) -> (Tensor, Tensor, Tensor, Tensor, Tensor)
```

Comment:
Not sure about this one. How/where is it called?

Reply:
This we call in the RNN.cpp.